css/_recording.scss

```diff
@@ -3,8 +3,23 @@
 }
 
 .recording-dialog {
+    flex: 0;
```

Comment:
I personally like the separation of commits a LOT more that the previous single commit! Thank you very much, @hristoterezov 

I'm afraid I'm scratching my head a bit here...

> This involves redesign of the web recording dialog in order to look the same as the mobile one.

I'm not at all sure why the redesign of the Web recording dialog exists in the commit which implements the mobile support. Both the Web and mobile dialog's are a done deal - design/Yana has requested them like that and both dialogs should be there anyway. If Saúl (read a dev) successfully argues for a JavaScript-only implementation (e.g. reverts the commit with the native Dropbox SDK), how is that going to affect the Web recording dialog's design? Why would it at all change anything in the UI/UX of both dialogs? Implementation details maybe but overall?

From a bird's eye view, I'd even say have the react-native dialog in a separate commit and the native Dropbox SDK in another one. Right now I get the feeling that the SDK is in the functions.web.js, functions.native.js, functions.any.js. I'm likely wrong but I'll be looking at it tomorrow so please don't waste your time anymore.

(I may still be a little dizzy right now from today's anesthesia.)